test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsControllerSpec.scala

```diff
@@ -1355,7 +1362,50 @@ class ReliefDetailsControllerSpec
         }
 
         checkIsRedirect(
-          performAction(),
+          performAction(Seq("privateResidentsRelief" -> "1")),
+          routes.ReliefDetailsController.otherReliefsSubmit()
```

Comment:
this is weird - why would it be redirecting to the submit endpoint? Is this test actually passing?

Reply:
The reason it is passing should be that submit also redirects to residents relief page in case that does not exist. The same is done on the display endpoint too. This is the same behaviour when it was the lettingsRelief. It makes more sense to make it otherReliefs() rather than otherReliefsSubmit() though as per the thing we are trying to mimic.

Reply:
Done.